### ISSUE TITLE
Favourites / individual folders / feature in asset browser

### ISSUE THREAD

**Issue by TobiasElemental** [OurMachinery/themachinery-public#1042]
It would be really great if there was a tab within the asset browser where people could create their "favourites" folders, or whatever they want to name it really. For easier and quicker access to specific things.

Often a project has a folder and naming structure, but individuals have their own preferences and different things they like to access frequently.

**Comment by simon-ourmachinery** [comment]
@tobias-ourmachinery  do you mean something like bookmarks in chrome? So basically a reference to different folder lets say:

`sm_assets` (static mesh assets) --> "renamed" (book marked) locally stored in e.g. settings as `my assets`

**Comment by tobias-ourmachinery** [comment]
@simon-ourmachinery I think you meant to direct the question to @TobiasElemental

**Comment by simon-ourmachinery** [comment]
Ah sorry I picked the wrong Tobias

**Comment by TobiasElemental** [comment]
@simon-ourmachinery yes pretty much that. so that individuals can have quick access to the things they need in various categorized folders only visible to them, not affecting projects folder and asset browser structure.

**Comment by simon-ourmachinery** [comment]
Ok do you have concrete example of a UX /UI ? Bookmarks in windows explorer or bookmarks in chrome?


What I am hearing is that you are missing quick access to customised things such as assets or folders? Does this also apply to other things in the engine?

**Comment by simon-ourmachinery** [comment]
I think in order to implement this we would need #1198 otherweise settings would be shared across the collab session.

Concepts:
-  bookmark/favorite view
-  Collection mechanism / view. (Kinda local labels)
